snapm/progress.py

```
@@ -793,27 +792,29 @@ def _do_start(self):
         """
         return  # pragma: no cover
 
-    def _do_progress(self, _done: int, _message: Optional[str] = None):
+    # pylint: disable=unused-argument
+    def _do_progress(self, done: int, message: Optional[str] = None):
```

Ruff (0.14.8)

Unused method argument: `done`

(ARG002)